src/Entity/Game.php

```diff
@@ -24,14 +24,14 @@ class Game
     private $id;
 
     /**
-     * @ORM\Column(type="string", length=255, nullable=true)
+     * @ORM\ManyToOne(targetEntity="Player", cascade={"all"}, fetch="EAGER")
```

Comment:
What is the `fetch="EAGER"` for?

Reply:
`EAGER` mean when I fetch a Game entity, `player1` will be directly initialized as Player entity.

Other can be `Lazy` and means that Doctirne will hydrate `player1` when I will call `$game>getPlayer1()`.
Second options should improve performance if I don't use everytime all relationships.

But If I always use this relationships, `Eager` is more indicated.